calm-ingest/mapping_collection.py

```diff
@@ -0,0 +1,60 @@
+# -*- encoding: utf-8 -*-
+"""
+Elasticsearch mapping for Collection records in Calm.
+"""
+
+import json
+
+import requests
+
+
+MAPPING = {
```

Comment:
Is there any mileage in just making these `.json` files and loading them? Might allow abstraction of the `push_mapping` below.